knockapi/client.py

```diff
@@ -20,6 +20,7 @@ def request(self, method, endpoint, payload=None):
         r = requests.request(
             method,
             url,
+            params=payload,
```

Comment:
parameters sent to GET endpoints were not working 😱

Reply:
😬

Reply:
@juanazam did you test POST payloads here as well? It seems a bit weird now that we're passing both `params=payload` and `json=payload`. I want to make sure this won't break anything.

Reply:
@cjbell will check so we are sure!

Reply:
@cjbell confirmed, tried both PUT and POST and everything goes through.